client/views/app.html

```diff
@@ -1,112 +1,187 @@
 </script>
-<link rel="stylesheet" href="../css/common.css" type="text/css" />
-<div id="app-container" ng-controller="GameController">
-  <div id="menu-container">
-    <div id="logo">
-      <span id="first-word">Cards</span>
-      <span id="second-word"> for</span>
-      <span id="third-word">Humanity</span>
-    </div>
-    <div id = 'abandon-game-button' ng-click="abandonGame()">
-        Abandon Game
-    </div>
-        <span id="tweet-container">
-      <a href="https://twitter.com/share" class="twitter-share-button" data-url="http://cfh.io" data-text="Cards for Humanity: A Game for Horrible People Desperately Trying to do Good" data-related="CFH_App">Tweet</a>
-    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
-    </span>
-      <div id="invite-container" ng-show="game.state === 'awaiting players'" >
-
-                  <div id="search-players">
-                <input
-                  type="search"
-                  placeholder="Search for users"
-                  name="search"
-                  ng-model="search"
-                  ng-change="searchUsers()"
-                >
-            </div>
-            <div id="invite-player" ng-show="game.state === 'awaiting players'">
-                    <div ng-repeat="player in match" style="display: flex">
-                      <span style="margin-right: 20px"> {{ player.name }}</span>
-                      <span style="margin-right: auto"> {{ player.email }}</span>
-              <span><button class= "" ng-click="inviteUsers(player.email)">Invite</button></span>
-                    </div>
+<link rel="stylesheet" href="../css/commo.css" type="text/css" />
```

Comment:
I noticed that this line is not importing the correct css file. If it's not in use, I suggest you remove the reference link completely.

Reply:
Thanks for the feedback. Will remove the file.

Reply:
Fixed!